doc/source/user/c-info.python-as-glue.rst

```diff
@@ -320,7 +320,7 @@ precision floating-point numbers using a fixed averaging filter. The
 advantage of using Fortran to index into multi-dimensional arrays
 should be clear from this example.
 
-.. code-block:: none
+.. code-block::
```

Comment:
```suggestion
.. code-block:: fortran
```

Reply:
There were a few blocks like this in the other docs too which don't seem to take to having syntax highlighting set.

Reply:
Got it - I was worried that removing the `none` would mean that the code-block would fall back to highlighting the block as if it were Python. I think this *is* the default behavior, though fortunately that doesn't matter for these blocks since everything is all caps :).

Thanks for the follow-up, LGTM then!